untitled.js

```diff
@@ -2727,3 +2114,118 @@ function handle_table(args, msg) {
         }
         });
 }
+
+
+function chatParser(msg) {
```

Comment:
I see what you're doing here, but could we instead get the roll template to call a macro "logCrit"?

Reply:
created #35